bower.json

```diff
@@ -8,13 +8,13 @@
   ],
   "dependencies": {
     "fxos-mvc": "git@github.com:fxos/mvc.git",
-    "gaia-theme": "gaia-components/gaia-theme",
-    "gaia-header" : "gaia-components/gaia-header",
-    "gaia-list": "gaia-components/gaia-list",
-    "gaia-button": "gaia-components/gaia-button",
-    "gaia-fonts": "gaia-components/gaia-fonts",
-    "gaia-dialog": "gaia-components/gaia-dialog",
-    "gaia-tabs": "gaia-components/gaia-tabs",
-    "gaia-sub-header": "gaia-components/gaia-sub-header"
+    "gaia-theme": "gaia-components/gaia-theme#0.4.5",
+    "gaia-header" : "gaia-components/gaia-header#0.7.0",
+    "gaia-list": "gaia-components/gaia-list#0.1.7",
+    "gaia-button": "git@github.com:mikehenrty/gaia-button.git#height-font-vars",
```

Comment:
I don't think we really need to fork this. ```gaia-button``` is pretty deficient, and your changes are very helpful and reasonable. We should fork the 'gaia-button' repo on the 'fxos' org, and make a PR for merging our changes when they're all ready. This will also help me as I'm using ```gaia-button``` in the Sharing app and running into similar problems.